Umbraco-Cloud/Set-Up/Manage-Hostnames/index.md

```diff
@@ -61,6 +61,10 @@ All hostnames added to an Umbraco Cloud project's environment will get a TLS (HT
 
 You will need to **remove the old DNS entry** before the Cloudflare service generates a new certificate for your Hostname.
 
+:::note
+Please note that the Certificate Authority (CA) used to issue certificates for all Umbraco Cloud sites' custom hostnames will change in [the coming months](ca-record-migration.md).
```

Comment:
The date (including the year) should be added to the note - "the coming months" is a little too relative ;)

I think it's also worth mentioning that: "No action is required unless you set a Certificate Authority Authorization (CAA) record on your domain." iN THE NOTE, SO OUR USERS DON'T HAVE TO CLICK THROUGH TO GET THAT INFORMATION.

Reply:
I've added the following to the note:

Please note that the Certificate Authority (CA) used to issue certificates for all Umbraco Cloud sites' custom hostnames will change for new and existing hostnames in [September and October 2022]